### DISCUSSION TITLE
Adding "table=True" on ORM entities removes the capability of pydantic to deserialize iso datetime string to python datetime.

### DISCUSSION THREAD

**Discussion by pboyadjian** [fastapi/sqlmodel#810] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Optional
from sqlmodel import SQLModel, Field


class MyClass(SQLModel, table=True):
  __tablename__ = "my_class"
  id: Optional[int] = Field(primary_key=True, default=None)
  timestamp: datetime
```

### Description

When adding property "table=True" on class argument pydantic does not deserialize the iso datetime string to python datetime.

### Operating System

Linux

### SQLModel Version

0.0.16

### Python Version

3.11

**Comment by matt-electronx** [comment]
Solution is here: https://stackoverflow.com/a/78057744
